tests/tools/test.sh

```diff
@@ -86,8 +86,8 @@ debug_command() {
 	fi
 	if (( depth <= DEBUG_LEVEL )); then
 		local indent=">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
-		echo "${indent:0:$depth} $(basename "${BASH_SOURCE[1]}"):${BASH_LINENO[0]}:`
-				`${FUNCNAME[1]}(): ${BASH_COMMAND}" >&2 | true
```

Comment:
wow what this double grave `` were here for?

Reply:
Indeed, this change does seem to affect at least this particular one:

```
+++test.sh:90:debug_command(): echo '>> timeout.sh:71:
				timeout_set(): timeout_set 2 minutes'
>> timeout.sh:71:
				timeout_set(): timeout_set 2 minutes
+++timeout.sh:72:timeout_set(): debug_command
```

I would still like to avoid using `, since they are hard for even other parsers to correctly detect (like from github in the issue above)

Reply:
Best solution in my opinion is just to make them variables instead, also improves readability:

```bash
		local indent=">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
		local level=${indent:0:$depth}
		local source=$(basename "${BASH_SOURCE[1]}")
		local line=${BASH_LINENO[0]}
		local function=${FUNCNAME[1]}
		local command=${BASH_COMMAND}
		echo "${level} ${source}:${line}: ${function}(): ${command}" >&2 | true
```